src/components/colorscale/color_attributes.js

```diff
@@ -64,14 +69,16 @@ module.exports = function makeColorScaleAttributes(context) {
         autocolorscale: extendDeep({}, colorScaleAttributes.autocolorscale, {
```

Comment:
@etpinard yes, tests support you, assuming you actually meant `cauto` (`autocolorscale` just toggles between a default palette vs. a palette determined by the user-supplied `colorscale`). 

In any case, our common conclusion is that the user does not _need_ to set both of `cmin` and `cmax`, yet the current doc says both are needed. I'll leave it as it is because, with the current helptext, partial `cmin` / `cmax` specification leads to *undefined* results, which is of course fine even if we handle it gracefully. I just wasn't sure if you wanted to keep this constraint in the helpstring, or it was a case of just not having updated the doc string.